.github/workflows/android-instrumentation-tests.yml

```diff
@@ -0,0 +1,50 @@
+name: Android Instrumentation Tests
+
+on:
+  workflow_dispatch: {}
```

Comment:
Can remove this since it's empty?

Reply:
Unfortunately `workflow_dispatch` must be defined to be able to dispatch a workflow through GitHub APIs which is what we do in `et dispatch` command. If we remove this, we wouldn't be able to trigger it that way.